server/src/utils.ts

```diff
@@ -156,6 +156,24 @@ export let runAnalysisAfterSanityCheck = (
   return JSON.parse(stdout.toString());
 };
 
+export let runAnalysis = (args: Array<any>) => {
+  let binaryPath;
+  if (fs.existsSync(c.analysisDevPath)) {
```

Comment:
This logic should not be duplicated.
There's a function called `getAnalysisBinaryPath`.

Reply:
This function was client side, add it to `utils.ts`